src/js/tracks/text-track-list.js

```diff
@@ -111,6 +111,9 @@ TextTrackList.prototype.removeTrack_ = function(rtrack) {
   for (let i = 0, l = this.length; i < l; i++) {
     if (this[i] === rtrack) {
       track = this[i];
+      if (track.off) {
```

Comment:
This works. Sorry, what I meant was checking to see if it's an `instanceof` `videojs.TextTrack`.